### DISCUSSION TITLE
The purpose of working_dir

### DISCUSSION THREAD

**Discussion by grantholle** [raphamorim/rio#165] [Q&A] [answered]
I've set my config's `working_dir` to be my home directory. When I open a new tab, it is whatever my previous tab's directory is. Am I not understanding what `working_dir` is setting?

**Comment by raphamorim** [comment]
Hey there @grantholle , which version are you using? I thought it was fixed in the main/canary.

**Reply by grantholle** [reply]
0.0.14, on Mac if it's relevant.

**Reply by raphamorim** [reply, marked as answer]
can you give a try with the canary version? https://github.com/raphamorim/rio/releases/tag/canary-26a22cae9aee813988d8c793fcb143e7d40f383a

If isn't fixed I will move this discussion to an issue, because if working_dir is settled up should be working_dir (unless navigation.use-current-path is true)

**Reply by grantholle** [reply]
Canary worked as expected
